From 4e1be45e92a5af6cef269eb4dd82e3e99bb71dd8 Mon Sep 17 00:00:00 2001
From: Adrian Alonso <aalonso@freescale.com>
Date: Wed, 1 Aug 2012 14:26:27 -0500
Subject: ENGR00219139: imx6 Sabreauto HDMI DDC I2C writing error

* imx6 Sabreauto HDMI DDC configuration causes I2C
  writing error
* Fix with HDMI, I2C2 SCL pad configuration for SABRE-AI
  MX6Q_PAD_EIM_EB2__HDMI_TX_DDC_SCL
  MX6Q_PAD_EIM_EB2__I2C2_SCL

Signed-off-by: Adrian Alonso <aalonso@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_sabreauto.h    |    4 ++--
 arch/arm/mach-mx6/board-mx6solo_sabreauto.h |    4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabreauto.h b/arch/arm/mach-mx6/board-mx6q_sabreauto.h
index da6e438..477078c 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabreauto.h
+++ b/arch/arm/mach-mx6/board-mx6q_sabreauto.h
@@ -361,11 +361,11 @@ static iomux_v3_cfg_t mx6q_weimnor_pads[] __initdata = {
 };
 
 static iomux_v3_cfg_t mx6q_sabreauto_hdmi_ddc_pads[] = {
-	MX6Q_PAD_KEY_COL3__HDMI_TX_DDC_SCL, /* HDMI DDC SCL */
+	MX6Q_PAD_EIM_EB2__HDMI_TX_DDC_SCL,  /* HDMI DDC SCL */
 	MX6Q_PAD_KEY_ROW3__HDMI_TX_DDC_SDA, /* HDMI DDC SDA */
 };
 
 static iomux_v3_cfg_t mx6q_sabreauto_i2c2_pads[] = {
-	MX6Q_PAD_KEY_COL3__I2C2_SCL,    /* I2C2 SCL */
+	MX6Q_PAD_EIM_EB2__I2C2_SCL,     /* I2C2 SCL */
 	MX6Q_PAD_KEY_ROW3__I2C2_SDA,    /* I2C2 SDA */
 };
diff --git a/arch/arm/mach-mx6/board-mx6solo_sabreauto.h b/arch/arm/mach-mx6/board-mx6solo_sabreauto.h
index f51925b..2e9e495 100644
--- a/arch/arm/mach-mx6/board-mx6solo_sabreauto.h
+++ b/arch/arm/mach-mx6/board-mx6solo_sabreauto.h
@@ -361,11 +361,11 @@ static iomux_v3_cfg_t mx6dl_weimnor_pads[] __initdata = {
 };
 
 static iomux_v3_cfg_t mx6dl_sabreauto_hdmi_ddc_pads[] = {
-	MX6DL_PAD_KEY_COL3__HDMI_TX_DDC_SCL, /* HDMI DDC SCL */
+	MX6DL_PAD_EIM_EB2__HDMI_TX_DDC_SCL,  /* HDMI DDC SCL */
 	MX6DL_PAD_KEY_ROW3__HDMI_TX_DDC_SDA, /* HDMI DDC SDA */
 };
 
 static iomux_v3_cfg_t mx6dl_sabreauto_i2c2_pads[] = {
-	MX6DL_PAD_KEY_COL3__I2C2_SCL,   /* I2C2 SCL */
+	MX6DL_PAD_EIM_EB2__I2C2_SCL,    /* I2C2 SCL */
 	MX6DL_PAD_KEY_ROW3__I2C2_SDA,   /* I2C2 SDA */
 };
-- 
1.7.9.5

